src/timestamp.cpp

```diff
@@ -0,0 +1,53 @@
+#include "g3log/time.hpp"
+#include "g3log/timestamp.hpp"
+#include <sstream>
+
+
+namespace g3 {
+
+
+	Timestamp::Timestamp(std::string& defaultFormat) :
```

Comment:
Use the posix struct. I'll do some digging. If support on Windows is bad then we at worst have to #ifdef it at: 
It's creation, the formatting function and the struct member itself (Windows : time_t, all other struct timeval)